README.md

```diff
@@ -2,28 +2,75 @@
 
 Merge Gatekeeper provides extra control for Pull Request management.
 
-## 🌄 What does Merge Gatekeeper provide, and Why?
+## 🌄 What Does Merge Gatekeeper Provide, and Why?
 
 <!-- == imptr: background / begin from: ./docs/details.md#[background] == -->
 
-Pull Request plays a significant part in day-to-day development, and making sure all the merges are controled is essential for building robust system. GitHub provides some control over CI, reviews, etc., but there are some limitations with handling special cases.
+Pull Request plays a significant part in day-to-day development, and it is essential to ensure all merges are well controlled and managed to build robust system. GitHub provides controls over CI, reviews, etc., but there are some limitations around handling specific use cases. Merge Gatekeeper helps overcome those by adding extra controls, such as monorepo friendly job check.
```

Comment:
"monorepo friendly job check" is vague to me and "monorepo friendly branch protection" might be better

Reply:
Good point, I'll update accordingly!

Reply:
This is updated with c018302